transport/http/src/main/java/org/kaazing/gateway/transport/http/HttpAcceptProcessor.java

```diff
@@ -226,6 +226,9 @@ public void operationComplete(WriteFuture future) {
 
     @Override
     protected void consume(DefaultHttpSession session) {
+        if (session.isReadSuspended()) {
+            return;
+        }
```

Comment:
It's called from org.kaazing.gateway.transport.http.HttpAcceptor.bridgeHandler.new IoHandlerAdapter() {...}.doMessageSent(IoSessionEx, Object) which does not have the guard. If you prefer I can put the if condition there instead.